File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int h;

    // get the height of the pyramid from the user
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (int i = 1; i <= h; i++) // loop for print the number of lines that will be the height of the pyramid
    {

        for (int x = h; x > i; x--) // print spaces
        {
            printf(" ");
        }

        for (int j = 0; j < i; j++) // loop for print the number of hashes in each line of the pyramid
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# Getting input from the user and checking it is valid
while True:
    input = get_int("Height is: ")
    if input <= 0 or input > 8:
        print("Height must be between 0 and 8.")
        continue
    else:
        break

# Printing pyramid

# Print the height of pyramid
for i in range(input):

    # Printing spaces
    for s in range(input - 1 - i):
        print(' ', end="")

    # Printing hashes
    for j in range(i + 1):
        print("#", end="")

    # Printing double space between pyramids
    print("  ", end="")

    # Printing second pyramid
    for h in range(i + 1):
        print("#", end="")

    print()
